src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -959,6 +1006,10 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
   glb_policy->lb_call_timeout_ms =
       grpc_channel_arg_get_integer(arg, (grpc_integer_options){0, 0, INT_MAX});
 
+  arg = grpc_channel_args_find(args->args, GRPC_ARG_GRPCLB_FALLBACK_TIMEOUT_MS);
+  glb_policy->lb_fallback_timeout_ms =
+      grpc_channel_arg_get_integer(arg, (grpc_integer_options){10000, 0, INT_MAX});
```

Comment:
Instead of hard-coding `10000` here, I suggest adding a macro for it at the top of the file:

```
#define GRPC_GRPCLB_DEFAULT_FALLBACK_TIMEOUT_MS 10000
```

Then use that macro here.

This approach makes it easier to see which defaults can be tuned without having to look throughout the whole code.

Reply:
Sure. 

I will remove the comments about "Default value is 10,000" too.